gsearchtool/src/gsearchtool-callbacks.c

```diff
@@ -663,6 +663,38 @@ open_folder_cb (GtkAction * action,
 	g_object_unref (g_app_info);
 }
 
+void
+copy_path_cb (GtkAction * action,
+                gpointer data)
+{
+	GSearchWindow * gsearch = data;
+	GtkTreeModel * model;
+	GtkTreeIter iter;
+	GList * list;
+	gchar * locale_path;
+	GtkClipboard* clipboard;
+
+	if (gtk_tree_selection_count_selected_rows (GTK_TREE_SELECTION (gsearch->search_results_selection)) == 0) {
```

Comment:
It should be:
```C
if (gtk_tree_selection_count_selected_rows (GTK_TREE_SELECTION (gsearch->search_results_selection)) != 1)
	return;
```
In any case, the number of rows is already checked:
```C
/* Popup menu item: Copy Path */
if (gtk_tree_selection_count_selected_rows (GTK_TREE_SELECTION (gsearch->search_results_selection)) == 1) {
```

Reply:
You're right.